apps/backend/src/modules/categories/category.service.ts

```diff
@@ -69,12 +75,32 @@ class CategoryService implements Service {
 	public async find(id: number): Promise<CategoryDto | null> {
 		const categoryEntity = await this.categoryRepository.find(id);
 
-		return categoryEntity
-			? this.convertCategoryEntityToDto(categoryEntity)
-			: null;
+		if (!categoryEntity) {
+			return null;
+		}
+
+		const { createdAt, name, updatedAt } =
+			this.convertCategoryEntityToDto(categoryEntity);
+
+		return { createdAt, id, name, updatedAt };
+	}
+
+	public async findAll(): Promise<GetCategoriesDto> {
+		const entities = await this.categoryRepository.findAll();
+
+		return {
+			items: entities.map((entity) => {
+				const { createdAt, id, name, updatedAt } =
+					this.convertCategoryEntityToDto(entity);
+
+				return { createdAt, id, name, updatedAt };
+			}),
+		};
 	}
 
-	public async findAll(): Promise<{ items: CategoryDto[] }> {
+	public async findAllWithScores(): Promise<{
```

Comment:
```suggestion
	public async findAll(): Promise<{
```

Reply:
But there will be mapping logic on the category controller if I revert this method.

Below are the result of `CategoryWithScoresDto[]`

![Screenshot (343)](https://github.com/user-attachments/assets/a10353ae-d2a6-4597-ba60-6e4c11f98c41)